pkg/image/transport.go

```diff
@@ -68,7 +68,11 @@ func buildSourceContext(accessKey, secKey, certDir string, insecureRegistry bool
 }
 
 func readImageSource(ctx context.Context, sys *types.SystemContext, img string) (types.ImageSource, error) {
-	ref, err := alltransports.ParseImageName(img)
+	parts := strings.SplitN(img, ":", 2)
```

Comment:
I think we should assert that parts[0] == "docker"

Reply:
Yeah the unit tests are failing because we use oci-archive (for tars) there. So we should support that one as well